### PR TITLE
Gui v2 switch

### LINKED ISSUE #616: Switch to GUI v2
- [x] Move v2 GUI repo to OpenEVSE
- [ ] Switch sub module to v2 GUI
- [ ] Remove v1 building from Workflows
- [x] Create v4 branch pre PR merge
- [x] Update docs (mostly done in https://github.com/OpenEVSE/ESP32_WiFi_V4.x/pull/623)

### PR DESCRIPTION
**Pull request by KipK**
Not sure I've missed something here.  Perhaps the github action key on openevse gui-v2 repo has to be set?

I've changed the gui directory to gui-v2 to prevent some git error I got if I keep the same path for different submodules

Fixes #616

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -59,13 +55,9 @@ jobs:
           - openevse_esp32-poe-iso
           - openevse_esp32-heltec-wifi-lora-v2
         gui:
-          - name: gui-v1
-            repo: OpenEVSE/openevse_wifi_gui
-            build_flags: ""
-            ref: ${{ inputs.v1_ref }}
           - name: gui-v2
-            repo: KipK/openevse-gui-v2
-            build_flags: "-D DISABLE_WIFI_PORTAL -D WEB_SERVER_ROOT_PAGE_INDEX"
```
Comment (jeremypoulter) [inline .github/workflows/build.yaml:68]: These need defining in the C code

**Review by jeremypoulter — COMMENTED** [review]
Should remove all the v1 header files as well, and I don't quite understand the reason for changing the folder to `gui-v2`

File: .github/workflows/build.yaml (hunk 1 above)
Comment (KipK) [reply .github/workflows/build.yaml:68]: Haven't I removed them on this pr ?

Comment (KipK) [reply .github/workflows/build.yaml:68]: ho it was on my other working branch. forgot to cherry pick it here

Comment (jeremypoulter) [reply .github/workflows/build.yaml:68]: I don't believe so, althopugh that is indeed the other option

Comment (KipK) [reply .github/workflows/build.yaml:68]: I've put it back should be good now

**Comment by KipK** [comment]
>  I don't quite understand the reason for changing the folder to `gui-v2`

I got some references errors on existing cloned repo when it switched to the other "gui" with different submodule repository.
It worked ok when you clone from scratch but not on existing ones when I try to switch back & forth from master and guiv2switch

**Review by jeremypoulter — APPROVED** [review]

**Pull request merged**